examples/website/line/app.js

```diff
@@ -43,89 +43,65 @@ function getSize(type) {
   return 60;
 }
 
-export default class App extends Component {
-  constructor(props) {
-    super(props);
-    this.state = {
-      hoveredObject: null
-    };
-    this._onHover = this._onHover.bind(this);
-    this._renderTooltip = this._renderTooltip.bind(this);
-  }
-
-  _onHover({x, y, object}) {
-    this.setState({x, y, hoveredObject: object});
-  }
-
-  _renderTooltip() {
-    const {x, y, hoveredObject} = this.state;
-    return (
-      hoveredObject && (
-        <div className="tooltip" style={{left: x, top: y}}>
-          <div>{hoveredObject.country || hoveredObject.abbrev}</div>
-          <div>{hoveredObject.name.indexOf('0x') >= 0 ? '' : hoveredObject.name}</div>
-        </div>
-      )
-    );
-  }
-
-  _renderLayers() {
-    const {
-      airports = DATA_URL.AIRPORTS,
-      flightPaths = DATA_URL.FLIGHT_PATHS,
-      getWidth = 3
-    } = this.props;
-
-    return [
-      new ScatterplotLayer({
-        id: 'airports',
-        data: airports,
-        radiusScale: 20,
-        getPosition: d => d.coordinates,
-        getFillColor: [255, 140, 0],
-        getRadius: d => getSize(d.type),
-        pickable: true,
-        onHover: this._onHover
-      }),
-      new LineLayer({
-        id: 'flight-paths',
-        data: flightPaths,
-        opacity: 0.8,
-        getSourcePosition: d => d.start,
-        getTargetPosition: d => d.end,
-        getColor,
-        getWidth,
-        pickable: true,
-        onHover: this._onHover
-      })
-    ];
-  }
-
-  render() {
-    const {mapStyle = 'mapbox://styles/mapbox/dark-v9'} = this.props;
-
-    return (
-      <DeckGL
-        layers={this._renderLayers()}
-        initialViewState={INITIAL_VIEW_STATE}
-        controller={true}
-        pickingRadius={5}
-        parameters={{
-          blendFunc: [GL.SRC_ALPHA, GL.ONE, GL.ONE_MINUS_DST_ALPHA, GL.ONE],
-          blendEquation: GL.FUNC_ADD
-        }}
-      >
-        <StaticMap
-          reuseMaps
-          mapStyle={mapStyle}
-          preventStyleDiffing={true}
-          mapboxApiAccessToken={MAPBOX_TOKEN}
-        />
+function getTooltip({object}) {
+  return object
```

Comment:
I have to say that while prettier has many advantages, I am not a fan of the way it butchers small ternary expressions like this. This is of course just my personal opinion, bit to me this detracts from the "elegance of expression" of the rest of the examples.

Adding a `prettier-ignore` comment would probably look too heavy handed, but I think rewriting it as `return object || {...` would make these functions format more cleanly in the examples.